liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/UndeployAppMojo.java

```diff
@@ -82,7 +82,7 @@ protected void doExecute() throws Exception {
                 .createTask("antlib:net/wasdev/wlp/ant:undeploy");
         
         if (undeployTask == null) {
-            throw new NullPointerException("Undeploy task not found");
+            throw new IllegalStateException("Undeploy task not found");
```

Comment:
Seeing the pattern here, so it probably would be good to add this to the messages file as:
The Liberty {0} Ant task was not found. Verify that the required version of the wlp-anttasks.jar is available.
Then you call the MessageFormat.format method with the message key and the task name.